packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -23,7 +23,7 @@
   transition-duration: 100ms;
   transition-timing-function: linear;
   border: 1px solid transparent;
-  background-color: $color-primary-base;
+  background-color: var(--ColorBackgroundPrimary);
   box-shadow: 0 0 0 0 $color-primary-light;
   color: $white;
```

Comment:
As we create tokens for all of the variables, will we need to create corresponding mappings in `Button.theme`?

If so, could we automate that via Style Dictionary in the future?

Reply:
yes we will. The mapping needs to be made explicit from global properties to component level properties. Definitely open to discussing ways to automate that process! Right now the fact that the mapping is constructed in a JS function (our reducer) means that we already have the ability to apply scripting to keep things DRY (building repeatable variants etc) which is good.

Reply:
@edburyenegren-okta revisiting this topic too, creating the mapping seems like something we may be able to do in style dictionary after playing with the API a bit today. It would require "lifting" all component token definitions into the separate package away from their styles, at least for ODS core components. Lets talk more at standup about benefits and tradeoffs here